src/main/java/com/mykaarma/urlshortener/exception/BadShorteningRequestException.java

```diff
@@ -0,0 +1,50 @@
+package com.mykaarma.urlshortener.exception;
+
+import com.mykaarma.urlshortener.dto.UrlErrorDTO;
+import com.mykaarma.urlshortener.enums.UrlErrorCodes;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class BadShorteningRequestException extends ShortUrlException{
+	
+	/** The Constant serialVersionUID. */
+	private static final long serialVersionUID = 1L;
+
+	/** The error DTO. */
+	private UrlErrorDTO errorDTO;
+	
+	/** The error data. */
+	private String errorData;
+
+	/**
+	 * Instantiates a new ShortUrlNotFound exception.
```

Comment:
@AadiS11 The comments got copied from ShortUrlNotFoundException from everywhere

Reply:
Fixed the comments